lib/crowbar/init/application.rb

```diff
@@ -256,6 +256,16 @@ def migrate_database
         false
       end
 
+      def migrate_crowbar
+        cmd = run_cmd(
+          "cd /opt/dell/crowbar_framework && " \
```

Comment:
Typically an array is only used for command execution if each item in the array is an argument in a single command.  In this case the command will get passed to a shell, so a single string makes more sense to me.